locales/en/moves.json

```diff
@@ -151,12 +151,16 @@
     }
   },
   "confirmation": {
-    "page_title": "Move scheduled for {{name}}",
+    "page_title_requested": "Move scheduled for {{name}}",
+    "page_title_proposed": "Move sent for {{name}}",
     "panel": {
-      "heading": "Move scheduled",
-      "content": "Reference number<br><strong>{{moveReference}}</strong>"
+      "heading_requested": "Move scheduled",
+      "heading_proposed": "Move sent",
+      "content_requested": "Reference number<br><strong>{{moveReference}}</strong>",
```

Comment:
We probably don't need to use different contexts for the reference number, we'll use the same text here regardless of status.